docs/content/configuration/csi_ug_config_create_statefulset.md

```diff
@@ -20,108 +20,108 @@ The `statefulset.apps/<filename> created` message is emitted.
+          containers:
+          - name: demo-container
+            image: registry.access.redhat.com/ubi8/ubi:latest
+            command: [ "/bin/sh", "-c", "--" ]
+            args: [ "while true; do sleep 30; done;" ]
+            <b>volumeMounts:
+              - name: demo-volume-file-system
+                mountPath: "/data"</b>
+          volumes:
           - name: demo-volume-file-system
-            mountPath: "/data"</b>
-      volumes:
-      - name: demo-volume-file-system
-        persistentVolumeClaim:
-          claimName: demo-pvc-file-system
-</pre>
+            persistentVolumeClaim:
+              claimName: demo-pvc-file-system
 
 ## Creating a StatefulSet with raw block volume
 
 Create a StatefulSet yaml file, similar to the following demo-statefulset-raw-block.yaml file.
 
-<pre>
-kind: StatefulSet
-apiVersion: apps/v1
-metadata:
-  name: demo-statefulset-raw-block
-spec:
-  selector:
-    matchLabels:
-      app: demo-statefulset
-  serviceName: demo-statefulset
-  replicas: 1
-  template:
+Here, the `volumeDevices` indicates both the name of the volume, with the necessary `devicePath` of `"/dev/block"`.
+
+    kind: StatefulSet
+    apiVersion: apps/v1
     metadata:
-      labels:
-        app: demo-statefulset
+      name: demo-statefulset-raw-block
     spec:
-      containers:
-      - name: demo-container
-        image: registry.access.redhat.com/ubi8/ubi:latest
-        command: [ "/bin/sh", "-c", "--" ]
-        args: [ "while true; do sleep 30; done;" ]
-        <b>volumeDevices:
+      selector:
+        matchLabels:
+          app: demo-statefulset
+      serviceName: demo-statefulset
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: demo-statefulset
+        spec:
+          containers:
+          - name: demo-container
+            image: registry.access.redhat.com/ubi8/ubi:latest
+            command: [ "/bin/sh", "-c", "--" ]
+            args: [ "while true; do sleep 30; done;" ]
+            volumeDevices:
+              - name: demo-volume-raw-block
+                devicePath: "/dev/block"
+          volumes:
           - name: demo-volume-raw-block
-            devicePath: "/dev/block"</b>
-      volumes:
-      - name: demo-volume-raw-block
-        persistentVolumeClaim:
-          claimName: demo-pvc-raw-block
-</pre>
+            persistentVolumeClaim:
+              claimName: demo-pvc-raw-block
 
 ## Creating a StatefulSet with both raw block and file system volumes
 
 Create a StatefulSet yaml file, similar to the following demo-statefulset-combined.yaml file.
 
-<pre>
-kind: StatefulSet
-apiVersion: apps/v1
-metadata:
-  name: demo-statefulset-combined
-spec:
-  selector:
-    matchLabels:
-      app: demo-statefulset
-  serviceName: demo-statefulset
-  replicas: 1
-  template:
+In a mixed file, it is important to indicate both the `volumeMounts` and  `volumeDevices` parameters, where `mountPath` is `"/data"` and `devicePath` is `"/dev/block"`.
```

Comment:
this phrasing makes it sound to me like these specific example values are important, as if `mountPath` has to be `/data` and `devicePath` has to be `/dev/block`

Reply:
@oriyarde  yes - is this not the case? This is how we present each one separately for filesystem and block.
Originally the section was bolded so it's important to point out section separately.
This is also how it was explained to me originally when we created the section

Reply:
I don't think this is exactly the case, I think there was a misunderstanding.
the user can choose whichever path they want inside their container, similar to a [docker mount destination](https://docs.docker.com/storage/bind-mounts/)

Reply:
DONE

Reply:
`both the `volumeMounts` and  `volumeDevices` parameters.` should be:
`both the `volumeMounts` and `volumeDevices` parameters.` (there's a double space after the `and`)

Reply:
DONE